WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -226,6 +241,7 @@ private boolean fetchUsersList(String dotComBlogId, final int localTableBlogId,
             @Override
             public void onSuccess(List<Person> peopleList, boolean isEndOfList) {
                 boolean isFreshList = (offset == 0);
+                mCanRefreshUsers = false;
```

Comment:
If I understand correctly, this flag is used as a way to limit the relevant data fetch to only happen once. It doesn't seem to have to do with capabilities, roles or permissions so, may I recommend renaming it to something like `mHasRefreshedUsers` (and adjust its semantics accordingly)? Same for the rest of `mCanRefresh*` vars. Thanks!